frontend-react/src/components/Sidebar.js

```diff
@@ -0,0 +1,89 @@
+import React  from 'react';
+import PropTypes from 'prop-types';
+import {Link} from 'react-router-dom'
+
+import {
+  Divider,
+  Drawer,
+  Grid,
+  List,
+  ListItem,
+  ListItemText
+} from '@material-ui/core';
+import {makeStyles} from "@material-ui/styles";
+import CloseIcon from '@material-ui/icons/Close';
+import IconButton from '@material-ui/core/IconButton';
+
+import LanguagePicker from './LanguagePicker';
+import routes from '../routes';
+import useT from '../utils/translation';
+import logo from '../assets/logo_codeforpoznan.svg';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    color: theme.palette.sidebar.fg,
+    background: theme.palette.sidebar.bg,
+    height: '100vh',
+    minWidth: 250,
+    padding: 16,
+  },
+  close: {
+    color: theme.palette.sidebar.fg,
+  },
+  link: {
+    color: theme.palette.sidebar.fg,
+    textDecoration: 'none',
+  },
+  grid: {
+    paddingTop: 16,
+  },
+}));
+
+const Sidebar = ({ open, onClose }) => {
+  const classes = useStyles();
+  const translations = {
+    // keys are taken from routes.js::routes[].key
+    Home: useT('Home page'),
+    Login: useT('Log in'),
+    Logout: useT('Log out'),
```

Comment:
To avoid capital letters  at the beginning I suggest to change keys in routes to lowercase.

Because this keys are used in two places now and there is a risk they can be changed, it would be nice to define a const to store possible values:
```
const ROUTES_KEYS = {
  HOME: 'home',
  ...
};
```
and use it both here and in routes, e.g. `[ROUTER_KEYS.HOME]: useT('Home page)`

Reply:
Thanks, will do that.

Also, small note - I tried to circumvent that problem by using `t()` function in `routes`, but it doesn't work for react... So the best bet is to define translations with `useT` hook, and that's what I did here.